mobile_config/migrations/20230708171204_add_pcs_key_role.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE key_role ADD VALUE IF NOT EXISTS 'pcs';-- Add migration script here
```

Comment:
“Add migration script here”?

Reply:
ha. that's the default comment the migration create script leaves; thought i'd highlighted and typed over it.